### PR TITLE
change a few configuration to get this to work in hugo latest version

### PR DESCRIPTION
**Pull request by ruddy-m**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by sheenarbw — CHANGES_REQUESTED** [review]
I worry that if people follow the readme to install hugo then things wont work.

Can you update the docs and the installation script as well?

**Comment by ruddy-m** [comment]
> I worry that if people follow the readme to install hugo then things wont work.
> 
> Can you update the docs and the installation script as well?

yeah i did want to invest the time on docs if this actually did not work on your machine or in the pipeline for some reason, so if its okay I can update the docs and the script

File: install_hugo.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-VERSION_HUGO="0.51"  # this version matches the one on AWS amplify
+VERSION_HUGO="0.111.3"  # this version matches the one on AWS amplify
```
Comment (ruddy-m) [inline install_hugo.sh:3]: not sure if this does match the version in AWS amplify how can I confirm that

Comment (ruddy-m) [reply install_hugo.sh:3]: @sheenarbw

**Pull request merged**